html-css/override-extend-variables/intro.md

```diff
@@ -1,4 +1,4 @@
-<strong>In this module, you will learn how to override and extend PatternFly css in the context of your application to accomodate your custom designs.</strong>
+<strong>In this module, you will learn how to override and extend PatternFly css in the context of your application to accommodate your custom designs.</strong>
```

Comment:
Can we change all instances of "you will learn how to.." to "Learn how to.." ? This way, we're following the Technical Writing Style Guide by sticking with present tense.